.github/pull_request_template.md

```diff
@@ -0,0 +1,33 @@
+Issues
+ My PR addresses the following Zookeeper issues and references them in the PR description:
+(#200 - Link your issue number here: You can write "Fixes #XXX". Please use the proper keyword so that the issue gets closed automatically. See https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue Any of the following keywords can be used: close, closes, closed, fix, fixes, fixed, resolve, resolves, resolved)
+
+Description
+ Here are some details about my PR, including screenshots of any UI changes:
+(Write a concise description including what, why, how)
+
+Tests
+ The following tests are written for this issue:
+(List the names of added unit/integration tests)
+
+The following is the result of the "mvn test" command on the appropriate module:
+(If CI test fails due to known issue, please specify the issue and test PR locally. Then copy & paste the result of "mvn test" to here.)
+
+Changes that Break Backward Compatibility (Optional)
+My PR contains changes that break backward compatibility or previous assumptions for certain methods or API. They include:
+(Consider including all behavior changes for public methods or API. Also include these changes in merge description so that other developers are aware of these changes. This allows them to make relevant code changes in feature branches accounting for the new method/API behavior.)
+
+Documentation (Optional)
+In case of new functionality, my PR adds documentation in the following wiki page:
+(Link the GitHub wiki you added)
+
+Commits
```

Comment:
I would consider removing this section, but still, it's your call. I'm okay either way